src/octokit-client.ts

```diff
@@ -4,8 +4,28 @@ import {
   restEndpointMethods,
   RestEndpointMethodTypes
 } from '@octokit/plugin-rest-endpoint-methods'
+import {HttpProxyAgent} from 'http-proxy-agent'
+import {HttpsProxyAgent} from 'https-proxy-agent'
```

Comment:
I think you might need to add these to package.json
See https://github.com/peter-evans/create-pull-request/blob/7380612b49221684fefa025244f2ef4008ae50ad/package.json#L37

Reply:
Thanks, Peter, I have added it.